packages/client/modules/teamDashboard/components/TeamTasksHeader/TeamTasksHeader.tsx

```diff
@@ -147,5 +213,12 @@ export default createFragmentContainer(TeamTasksHeader, {
       }
       ...TeamDashTeamMemberMenu_team
     }
+  `,
+  viewer: graphql`
```

Comment:
can put the `team` fragment inside this one so we only have to worry about a single prop